pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeConsumptionRateManager.java

```diff
@@ -87,15 +87,25 @@ public ConsumptionRateLimiter createServerRateLimiter(PinotConfiguration serverC
     double serverRateLimit =
         serverConfig.getProperty(CommonConstants.Server.CONFIG_OF_SERVER_CONSUMPTION_RATE_LIMIT,
             CommonConstants.Server.DEFAULT_SERVER_CONSUMPTION_RATE_LIMIT);
+    _serverRateLimiter = createServerRateLimiter(serverRateLimit, serverMetrics);
+    return _serverRateLimiter;
+  }
+
+  private ConsumptionRateLimiter createServerRateLimiter(double serverRateLimit, ServerMetrics serverMetrics) {
     if (serverRateLimit > 0) {
-      LOGGER.info("Set up ConsumptionRateLimiter with rate limit: {}", serverRateLimit);
+      LOGGER.info("Set up ConsumptionRateLimiter with rate limit: {}.", serverRateLimit);
```

Comment:
(minor) We don't usually put `.` at the end of log. People might mix it with decimal point. Same for other places